src/TelemetryChannels/ServerTelemetryChannel/Shared/Implementation/BreezeResponse.cs

```diff
@@ -0,0 +1,30 @@
+﻿namespace Microsoft.ApplicationInsights.WindowsServer.TelemetryChannel.Implementation
+{
+    using System.Runtime.Serialization;
+
+    [DataContract]
+    internal class BreezeResponse
+    {
+        [DataMember(Name = "itemsReceived")]
+        public int ItemsReceived { get; set; }
+
+        [DataMember(Name = "itemsAccepted")]
+        public int ItemsAccepted { get; set; }
+
+        [DataMember(Name = "errors")]
+        public Error[] Errors { get; set; }
+
+        [DataContract]
+        public class Error
```

Comment:
The outer class BreezeResponse has the matching file name but there is an inner class named Error which is public.